### ISSUE TITLE
Breaking change introduced with 1.64.0 and add_registered_method_handlers function

### ISSUE THREAD

**Issue by prashantgupta24** [grpc/grpc#36683]
### What version of gRPC and what language are you using?
 1.64.0/python

### What operating system (Linux, Windows,...) and version?
linux

### What runtime / compiler are you using (e.g. python version or version of gcc)
python 3.9

### What did you do?
We are unable to create subclasses from `grpc.Server` anymore unless we additionally add the `add_registered_method_handlers` function, which was added as a new abstract method within `src/python/grpcio/grpc/__init__.py` in https://github.com/grpc/grpc/pull/36371

### What did you expect to see?

We should be allowed to inherit from `grpc.Server` without having to change existing behavior since this is a minor release?

### What did you see instead?

```
TypeError: Can't instantiate abstract class xyz with abstract method add_registered_method_handlers
```

See [TROUBLESHOOTING.md](https://github.com/grpc/grpc/blob/master/TROUBLESHOOTING.md) for how to diagnose problems better.

**Comment by gnossen** [comment]
Thanks for the report! This was an oversight -- these methods should not have been marked as abstract. We'll release a patch with this updated to resolve.

**Comment by XuanWang-Amos** [comment]
Reopen since we still need to backport this change.

**Comment by jvolkman** [comment]
It looks like this fix was released in 1.64.1 late last week. Will that version also be released to [pypi](https://pypi.org/project/grpcio/#history)? Thanks!

**Comment by XuanWang-Amos** [comment]
Yes, I just uploaded 1.64.1 with this fix.

With that, we're closing this issue.
